cedar-policy-core/src/entities/json/err.rs

```diff
@@ -127,8 +131,9 @@ pub enum JsonDeserializationError {
     #[error(transparent)]
     #[diagnostic(transparent)]
     ReservedName(#[from] ReservedNameError),
-    /// Returned when entities have tags, but the `entity-tags` feature is not enabled
-    #[cfg(not(feature = "entity-tags"))]
+    /// Never returned as of 4.2.0 (entity tags are now stable), but this error
+    /// variant not removed because that would be a breaking change on this
+    /// publicly-exported type.
```

Comment:
Does it make sense to deprecate enum variants?

Reply:
Added a deprecation